tests/test_deprecation.py

```diff
@@ -87,5 +87,4 @@ def test_deprecation_warnings_have_been_removed_after_two_minor_versions(
 
     # Chetan - 20220316 - Note that this will break as soon as v0.16.0 lands;
     # this should be cleaned up and made 0 at that point.
-    # Will - 20220414 - This was adjusted from 30 to <= 35 to unblock release. To be investigated further.
```

Comment:
Deleting this comment as this issue was caused by `dgtest` not picking up new deprecation warnings we added in. The change to CI/CD should catch future instances of this.